provider/database.py

```diff
@@ -37,9 +37,11 @@ class Database:
 
     filters_design_doc_id = '_design/filters'
     only_triggers_view_id = 'only-triggers'
+    by_worker_view_id = 'by-worker'
 
     instance = os.getenv('INSTANCE', 'messageHubTrigger-0')
-    canaryId = "canary-{}".format(instance)
+    workerId = os.getenv('WORKER', 'worker0')
```

Comment:
Looks like the call to determine the worker occurs in service.py. Instead value should be determine once and shared between the two classes.